Can't run and build on windows os

What happened: cann't build and run /debug in windows os

Environment:

Grafana version: v7.2.1
Data source type & version: none
OS Grafana is installed on: windows 10
User OS & Browser:
Grafana plugins: none
Others: 

"make run" command is not supports on windows , replace the command for  "mingw32-make run"
the error message is ：
```
Can't found file - *.sh
mingw32-make: *** [Makefile:50: scripts/go/bin/bra] Error 1
```

execute command "go run build.go build"
the following error:
```
Version: 7.2.2, Linux Version: 7.2.2, Package Iteration: 1603783139
rm -r ./bin/windows-amd64/grafana-server.exe
rm -r ./bin/windows-amd64/grafana-server.exe.md5
go version
go version go1.15.1 windows/amd64
Targeting windows/amd64
go build -ldflags -w -X main.version=7.2.2 -X main.commit=ad9d408ac2 -X main.buildstamp=1603265219 -X main.buildBranch=shizhuang-inc/v7.2.x -o ./bin/windows-amd64/grafana-server.exe ./pkg/cmd/grafana-server
# github.com/grafana/grafana/pkg/cmd/grafana-server
D:\Go\pkg\tool\windows_amd64\link.exe: running gcc failed: exit status 1
D:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: Error: export ordinal too large: 88685
collect2.exe: error: ld returned 1 exit status

exit status 2
exit status 1

```

then i want to debug in IDE ,  run main function in pkg/cmd/grafana-server/main.go, the error is always like this:
```
# github.com/grafana/grafana/pkg/cmd/grafana-server
D:\Go\pkg\tool\windows_amd64\link.exe: running gcc failed: exit status 1
D:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: Error: export ordinal too large: 84315
collect2.exe: error: ld returned 1 exit status
```

run grafana-cli ,it still report errors:
```
API server listening at: [::]:52478
NAME:
   Grafana CLI - A new cli application

USAGE:
   ___go_build_github_com_grafana_grafana_pkg_cmd_grafana_cli.exe [global options] command [command options] [arguments...]

VERSION:
   master

AUTHOR:
   Grafana Project <hello@grafana.com>

COMMANDS:
   plugins  Manage plugins for grafana
   admin    Grafana admin commands
   help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --pluginsDir value       Path to the Grafana plugin directory (default: "../data/plugins") [%GF_PLUGIN_DIR%]
   --repo value             URL to the plugin repository (default: "https://grafana.com/api/plugins") [%GF_PLUGIN_REPO%]
   --pluginUrl value        Full url to the plugin zip file instead of downloading the plugin from grafana.com/api [%GF_PLUGIN_URL%]
   --insecure               Skip TLS verification (insecure) (default: false)
   --debug                  Enable debug logging (default: false)
   --configOverrides value  Configuration options to override defaults as a string. e.g. cfg:default.paths.log=/dev/null
   --homepath value         Path to Grafana install/home path, defaults to working directory
   --config value           Path to config file
   --help, -h               show help (default: false)
   --version, -v            print the version (default: false)

Debugger finished with exit code 0

```

--- a/build.go
+++ b/build.go
@@ -482,6 +482,10 @@ func doBuild(binaryName, pkg string, tags []string) {
 		rmr(binary, binary+".md5")
 	}
 	args := []string{"build", "-ldflags", ldflags()}
+	if goos == windows {
+		// Work around a linking error on Windows: "export ordinal too large"
+		args = append(args, "-buildmode=exe")
+	}
 	if len(tags) > 0 {
 		args = append(args, "-tags", strings.Join(tags, ","))
 	}

--- a/build.go
+++ b/build.go
@@ -451,7 +451,12 @@ func gruntBuildArg(task string) []string {
 }
 
 func setup() {
-	runPrint("go", "install", "-v", "./pkg/cmd/grafana-server")
+	args := []string{"install", "-v"}
+	if goos == windows {
+		args = append(args, "-buildmode=exe")
+	}
+	args = append(args, "./pkg/cmd/grafana-server")
+	runPrint("go", args...)
 }
 
 func printGeneratedVersion() {
@@ -460,7 +465,12 @@ func printGeneratedVersion() {
 
 func test(pkg string) {
 	setBuildEnv()
-	runPrint("go", "test", "-short", "-timeout", "60s", pkg)
+	args := []string{"test", "-short", "-timeout", "60s"}
+	if goos == windows {
+		args = append(args, "-buildmode=exe")
+	}
+	args = append(args, pkg)
+	runPrint("go", args...)
 }
 
 func doBuild(binaryName, pkg string, tags []string) {

